### PR TITLE
Image stretching fix

### LINKED ISSUE #232: Image ratio changes when stretched
**Describe the bug**
Using the Image component directly, the ratio changes when stretched:

|Wide|Narrow|
|----|--------|
|screen shot 2018-12-19 at 10 43 34|screen shot 2018-12-19 at 10 43 40|

See https://bbc-news.github.io/psammead/?selectedKind=Image&selectedStory=default&full=0&addons=1&stories=1&panelRight=1&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

This is fixed when the Image is put inside an ImagePlaceholder - but we should fix the Image behaviour itself.

**Expected behavior**
Image to behave like the 16x9 with ImagePlaceholder: https://bbc-news.github.io/psammead/?selectedKind=ImagePlaceholder&selectedStory=16x9%20image%20placeholder&full=0&addons=1&stories=1&panelRight=1&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

I.e. proportionally scale down.

- [x] Initially labelled with ["bug"](https://github.com/BBC-News/psammead/labels/bug)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #232

Fixes the stretching issue by making the height prop optional.

Screenshot of the image scaling correctly with smaller breakpoints:
http://localhost:8080?selectedKind=Image
screen shot storybook image

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-image/CHANGELOG.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 | Version | Description |
 |---------|-------------|
+| 0.2.0 | [PR#](https://github.com/BBC-News/psammead/pull/) Change height prop to be optional. Update the storybook example. |
```
Comment (radiocontrolled) [inline packages/components/psammead-image/CHANGELOG.md]:
```suggestion
 | 0.2.0 | [PR#234](https://github.com/BBC-News/psammead/pull/234) Change height prop to be optional. Update the storybook example. |
```

File: packages/components/psammead-image/CHANGELOG.md [hunk 2]
```diff
@@ -2,4 +2,5 @@
 
 | Version | Description |
 |---------|-------------|
+ | 0.2.0 | [PR#234](https://github.com/BBC-News/psammead/pull/234) Change height prop to be optional. Update the storybook example. |
```
Comment (ChrisBAshton) [inline packages/components/psammead-image/CHANGELOG.md]:
```suggestion
| 0.2.0 | [PR#234](https://github.com/BBC-News/psammead/pull/234) Change height prop to be optional. Update the storybook example. |
```

**Review by ChrisBAshton — CHANGES_REQUESTED** [review]
Can we update the README and make it clear the height is optional (and the implications for specifying it)? https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-image#usage

File: packages/components/psammead-image/README.md [hunk 3]
```diff
@@ -13,12 +13,14 @@ const Wrapper = (src, alt, width, height) => (
...
-| `src`    | string        |
-| `width`  | number/string |
+| Prop     | Type          | Required |
+|:---------|:--------------|:---------|
+| `alt`    | string        | Yes      |
+| `height` | number/string | No       |
+| `src`    | string        | Yes      |
+| `width`  | number/string | Yes      |
+
+The `height` prop is optional, since in some cases to preserve the image ratio we only want to specify the width and let the browser scale the image accordingly. However, in other cases the height might need to be specified.
```
Comment (ChrisBAshton) [inline packages/components/psammead-image/README.md:23]: Nice 👏

**Review by ChrisBAshton — DISMISSED** [review]

**Review by radiocontrolled — DISMISSED** [review]

**Review by radiocontrolled — APPROVED** [review]
🐻

**Review by bcmn — APPROVED** [review]

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**